src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -1853,6 +1853,9 @@
 
   <Node Name="LocalDeclarationStatementSyntax" Base="StatementSyntax">
     <Kind Name="LocalDeclarationStatement"/>
+    <Field Name="UsingKeyword" Type="SyntaxToken" Optional="true">
+      <Kind Name="UsingKeyword"/>
+    </Field>
     <Field Name="Modifiers" Type="SyntaxList&lt;SyntaxToken&gt;">
```

Comment:
>Modifiers [](start = 17, length = 9)

I see that the parsing logic tolerates putting modifiers on local declarations. Can you test such scenarios with `using` locals?
I assume all modifiers in this position should be errors. Is that correct?

Reply:
Well, I would assume they would be binding errors, not syntax errors.

Reply:
Looks like we still need a test for this case.

I think we should test all of the following cases:

```C#
using ref int x = ref y;
using ref readonly int x = ref y;
using ref var x = ref y;
using ref var x = y;
using readonly var x, y = ref z;
```